### ISSUE TITLE
Automated deployment erroring out

### ISSUE THREAD

**Issue by taschmidt** [projectkudu/KuduScript#55]
I have 2 Azure web apps running on the same app service.  Setting up automated deployment works fine for one of them but not the other. I get the following error. Any ideas?

```
Using the following command to generate deployment script: 'azure site deploymentscript -y --no-dot-deployment -r "D:\home\site\repository" -o "D:\home\site\deployments\tools" --aspNetCore "D:\home\site\repository\src\{app}.Site" --solutionFile D:\home\site\repository\{app}.sln'.
Project file path: .\src\{app}.Site
error: Object .bowerrc has no method 'endsWith'
error: Object .bowerrc has no method 'endsWith'\r\nD:\Program Files (x86)\SiteExtensions\Kudu\58.50929.2438\bin\Scripts\starter.cmd "D:\Program Files (x86)\SiteExtensions\Kudu\58.50929.2438\bin\node_modules\.bin\kuduscript.cmd" -y --no-dot-deployment -r "D:\home\site\repository" -o "D:\home\site\deployments\tools" --aspNetCore "D:\home\site\repository\src\{app}.Site" --solutionFile D:\home\site\repository\{app}.sln

```

**Comment by davidebbo** [comment]
Odd. Could you share a [sample repo](https://github.com/projectkudu/kudu/wiki/Using-a-git-repo-to-report-an-issue) that demonstrates the issue? Thanks!

**Comment by davidebbo** [comment]
Actually, we know the issue, and have a PR for a fix: #54.

Can you check what `WEBSITE_NODE_DEFAULT_VERSION` is set to in your Web App? Try setting it to a newer version like `4.6.0` and that should solve it.

**Comment by taschmidt** [comment]
That fixed it!  It was set to "0.10.32" before.  Thanks!

I had never even paid attention to that setting before.  Are we supposed to keep that updated?

**Comment by davidebbo** [comment]
Well, that's up to you. Sometimes, you want to stick to a specific version to you're not affected by potential breaking changes.

Closing as the fix is now in, and will be deployed in a few days.
